src/main/java/org/prebid/server/bidder/BidderCatalog.java

```diff
@@ -112,6 +113,18 @@ public boolean isValidAdapterName(String name) {
         return bidderDepsMap.containsKey(name) && adapterByName(name) != null;
     }
 
+    /**
+     * Returns a bidder name to be used for metrics. In case given name is invalid and there's no alias for it -
+     * "UNKNOWN" is returned.
+     */
+    public String metricsBidderName(String bidder) {
```

Comment:
i'd like to see this functionality rather in `org.prebid.server.metric.Metrics` class.
`BidderCatalog` should not care about metric/bidder name.